src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplateDeleteItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Trans, useTranslation } from 'react-i18next';
+
+import { Modal } from 'shared/components';
+import { StyledBodyLarge, StyledModalWrapper } from 'shared/styles';
+
+type Props = {
+  itemToDelete: {
+    item: number;
+    shouldDelete: boolean;
+    fieldName?: string;
+  };
+  handleCloseDeleteModal: () => void;
+  handleOnSubmitDeleteModal: () => void;
+};
+
+function PhrasalTemplateDeleteItem({
+  itemToDelete,
+  handleCloseDeleteModal,
+  handleOnSubmitDeleteModal,
+}: Props): JSX.Element | null {
+  const { t } = useTranslation('app');
+
+  if (!itemToDelete || !itemToDelete.shouldDelete) return null;
+
+  const { fieldName } = itemToDelete;
+
+  return (
+    <Modal
+      open={!!itemToDelete.shouldDelete}
+      onClose={handleCloseDeleteModal}
+      onSubmit={handleOnSubmitDeleteModal}
+      onSecondBtnSubmit={handleCloseDeleteModal}
+      title={t('deleteItem')}
+      buttonText={t('delete')}
+      secondBtnText={t('cancel')}
+      hasSecondBtn
+      submitBtnColor="error"
+      data-testid="builder-activity-items-delete-item-popup"
+    >
+      <StyledModalWrapper>
+        <StyledBodyLarge sx={{ mb: 2.4 }}>
+          <Trans i18nKey="deleteItemDescription">
+            Are you sure you want to delete the Item
+            <strong>
+              <>{{ itemName: fieldName }}</>
+            </strong>
+            ?
+          </Trans>{' '}
+        </StyledBodyLarge>
+      </StyledModalWrapper>
+    </Modal>
+  );
+}
+
+export default PhrasalTemplateDeleteItem;
```

Comment:
Where are you seeing in Figma a delete dialogue? 🤔  The task AC doesn't actually mention a dialogue/popup/modal. This is what it says:

> When clicking the Delete button, a pop-up menu appears with a “🗑️ Remove” menu item, which removes the Text Input from the Phrase

Here's the [Delete interaction as shown in Figma](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=650-66873&t=stahY8y9jcvEC7d5-4):

![CleanShot 2024-08-22 at 19 14 53@2x](https://github.com/user-attachments/assets/96e3cfef-b8c5-4386-bc5d-0d75c003927f)
I'm curious where you saw that a popup/dialogue was the expected behaviour for the Trash icon. Perhaps there's another outdated version of the designs that I haven't seen – if so, could you share a link? I want to make sure there's no ambiguity in the future. Thanks!